.drone.yml

```diff
@@ -124,6 +124,9 @@ steps:
 
   - name: docker-publish
     image: rancher/drone-images:docker-s390x
+    volumes:
+      - name: docker
+        path: /var/run/docker.sock
```

Comment:
Why do we need the volume?  I do not see it is used in other docker-publish jobs for other arches.

Reply:
The other `docker-publish` jobs use a `plugin` image. Since this job doesn't use a plugin image, then then volume is required.

Reply:
ok, it makes sense. thanks for the explanation.